wasm-runtime/polkadot/src/primitives/header.rs

```diff
@@ -37,6 +37,18 @@ pub struct Header {
 	pub digest: Digest,
 }
 
+impl Header {
+	pub fn from_block_number(number: BlockNumber) -> Self {
```

Comment:
This is for testing only, right? A doc comment would be ince

Reply:
not really - useful in general when testing transactions without a specific preconstructed block to execute them within. doc comment sure